auth_backends/backends.py

```diff
@@ -100,8 +100,7 @@ def auth_complete(self, *args, **kwargs):
         return user
 
     def user_data(self, access_token, *args, **kwargs):
-        decoded_access_token = jwt.decode(access_token, verify=False)
-
+        decoded_access_token = jwt.decode(access_token, algorithms=["HS256"], options={"verify_signature": False})
```

Comment:
I am not sure from where to pick this value.

Reply:
By default [algorithm='HS256'](https://github.com/jpadilla/pyjwt/blob/1.7.1/jwt/api_jwt.py#L43) in `pyjwt==1.7.1`.

Reply:
with this `"verify_signature": False` it will just decode the payload, and not verify the signature.

Reply:
If this is accurate, would you mind adding the following comment?:
```
# The algorithm is required but unused because signature verification is skipped.
# Note: signature verification happens earlier during the authentication process.
```

Reply:
https://pyjwt.readthedocs.io/en/latest/usage.html#reading-the-claimset-without-validation